src/frontend/Komponenter/Behandling/Tabell/TabellVisning.tsx

```diff
@@ -1,81 +1,77 @@
 import * as React from 'react';
-import { Registergrunnlag, Søknadsgrunnlag } from '../../../Felles/Ikoner/DataGrunnlagIkoner';
-import { GridTabell } from '../../../Felles/Visningskomponenter/GridTabell';
-import { Calculator } from '@navikt/ds-icons';
-import { Heading, Label } from '@navikt/ds-react';
-import {
-    BodyShortSmall,
-    DetailSmall,
-    SmallTextLabel,
-} from '../../../Felles/Visningskomponenter/Tekster';
-
-export enum TabellIkon {
-    REGISTER = 'REGISTER',
-    SØKNAD = 'SØKNAD',
-    KALKULATOR = 'KALKULATOR',
-}
-
-export interface Kolonndata<T> {
-    ikon?: TabellIkon;
-    tittel: string;
-    undertittel?: string;
-    tittelType?: 'element' | 'undertittel';
+import { Detail, HelpText, Label } from '@navikt/ds-react';
+import { BodyShortSmall } from '../../../Felles/Visningskomponenter/Tekster';
+import styled from 'styled-components';
+import { mapIkon, VilkårInfoIkon } from '../Vilkårpanel/VilkårInformasjonKomponenter';
+export interface Kolonnedata<T> {
+    ikon?: VilkårInfoIkon;
+    tittel?: string;
     verdier: T[];
     kolonner: Kolonner<T>[];
+    ikonVisning?: boolean;
 }
 
 export interface Kolonner<T> {
     overskrift: string;
+    helperText?: string;
     tekstVerdi: (data: T) => React.ReactNode;
 }
 
-const mapIkon = (ikon: TabellIkon) => {
-    switch (ikon) {
-        case TabellIkon.REGISTER:
-            return <Registergrunnlag />;
-        case TabellIkon.SØKNAD:
-            return <Søknadsgrunnlag />;
-        case TabellIkon.KALKULATOR:
-            return <Calculator />;
+const GridTabell = styled.div<{
+    kolonner?: number;
+    ikonVisning: boolean;
+}>`
+    display: grid;
+    grid-template-columns: ${(props) => props.ikonVisning && '21px'} min(200px, 250px) repeat(
+            ${(props) => (props.kolonner ? props.kolonner - 2 : 2)},
+            ${(props) => (props.kolonner && props.kolonner > 3 ? '150px' : '325px')}
+        );
+    grid-gap: 0.5rem;
+
+    .tittel {
+        padding-bottom: 0.25rem;
+        grid-column: ${(props) => (props.ikonVisning ? 2 : 1)} /
+            ${(props) => (props.kolonner || 3) + 1};
+
+        display: flex;
+        align-items: center;
     }
-};
 
-function TabellVisning<T>(props: Kolonndata<T>): React.ReactElement<Kolonndata<T>> {
-    const { ikon, tittel, undertittel, tittelType, verdier, kolonner } = props;
+    .førsteDataKolonne {
+        grid-column: ${(props) => (props.ikonVisning ? '2/3' : '1/2')};
+    }
+`;
+
+const FlexDiv = styled.div`
+    display: flex;
+    gap: 1rem;
+`;
+
+function TabellVisning<T>(props: Kolonnedata<T>): React.ReactElement<Kolonnedata<T>> {
+    const { ikon, tittel, verdier, kolonner, ikonVisning = true } = props;
     return (
-        <GridTabell kolonner={kolonner.length + 1} utenIkon={!ikon}>
+        <GridTabell kolonner={kolonner.length + 1} ikonVisning={ikonVisning}>
```

Comment:
ikonVisning brukes i stedet for `utenIkon={!ikon}` fordi noen steder ønsker man å beholde innrykket som et ikon ville skapt selvom man ikke sender inn et ikon